decidim-core/spec/models/decidim/ability_spec.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+require "spec_helper"
+
+module Decidim
+  describe Ability do
+    let(:user) { build(:user) }
+    let(:abilities) { [] }
+
+    subject { described_class.new(user) }
+
+    class TestAbility
+      include CanCan::Ability
+
+      def initialize(user)
+        can :read, Decidim::ParticipatoryProcess
+      end
+    end
+
+    before do
+      allow(Decidim)
+        .to receive(:abilities)
+        .and_return(abilities)
+    end
+
+    it "does not hold any special ability" do
+      expect(subject.permissions[:can]).to be_empty
```

Comment:
Here I'm testing that there's no ability set, so I don't think I can use the helper method. In the other case I will, though.